docs/content/foundation/typography/text-styles.mdx

```diff
@@ -4,10 +4,56 @@ nav_label: Text Styles
 nav_order: 20
 ---
 
+import {
+	FontWeightTable,
+	FontWeightRow,
+} from '../../../src/components/typography';
+
 # NCIDS Text Styles
 
-Nulla facilisis vitae lectus quis ultricies. Aliquam sodales scelerisque sapien et dictum.
-Mauris eu nunc sit amet lorem blandit gravida et ac dui. Suspendisse accumsan eros sit
-amet arcu lobortis, in consequat nunc molestie. Aenean lobortis molestie convallis.
-Nullam sit amet turpis vel nisi tristique dapibus sed in libero.
-Vivamus ac tristique nisl, eu bibendum eros.
+Change the appearance of text. NCIDS uses Poppins Regular, Semibold, and Italics for headings. Semibold is used for Bold and Italics can be used for headings. Open Sans Regular and Bold are used for body text and in components.
+
+## Font Weight Tokens
```

Comment:
Why are we only including font weights?

https://designsystem.digital.gov/utilities/text-styles/

Reply:
the tables look awkward (small) but everything I could validate we had
https://designsystem-dev.cancer.gov/pr-1267/foundation/typography/text-styles